.about.yml

```diff
@@ -0,0 +1,111 @@
+---
+# .about.yml project metadata
+#
+# Short name that acts as the project identifier (required)
+name: useiti-report
+
+# Full proper name of the project (required)
+full_name: The United States Extractives Industries Transparency Initiative (USEITI)
+
+# The type of content in the repo
+# values: app, docs, policy
+type: app
+
+# Describes whether a project team, working group/guild, etc. owns the repo (required)
+# values: guild, working-group, project
+owner_type: project
+
+# Name of the main project repo if this is a sub-repo; name of the working group/guild repo if this is a working group/guild subproject
+#parent:
+
+# Maturity stage of the project (required)
+# values: discovery, alpha, beta, live
+stage: beta
+
+# Whether or not the project is actively maintained (required)
+# values: active, deprecated
+status: active
+
+# Description of the project
+description: |
+  This site and open data portal supports the President’s Open Gov Partnership National Action Plan commitment to the Extractive Industries Transparency Initiative.
+
+# Should be 'true' if the project has a continuous build (required)
+# values: true, false
+testable: true
+
+# Team members contributing to the project (required)
+# Items:
+# - github: GitHub user name
+#   id: Internal team identifier/user name
+#   role: Team member's role; leads should be designated as 'lead'
+team:
+- id: mhz
+  github: meiqimichelle
+  role: 18F Product Lead
+- id: shawn
+  github: shawnbot
+  role: Developer
+- id: brian
+  github: gemfarmer
+  role: Developer
+- id: ericronne
+  github: ericronne
+  role: 18F Designer, UX
```

Comment:
Sounds good, I'll do that and file an issue with 18F/about_yml to start the gears working on how we might implement partner team members.